### PR TITLE
Feature/bills issues

### PR DESCRIPTION
**Pull request by rodolforpr88**
Refactor some issues in the bills

add `status` field for the `bill`
add the validation `duplicated` to avoid duplicated calls if bill is billing or billed
refactor the test for `bill_spec`

### CODE REVIEW

File: app/models/bill.rb [hunk 1]
```diff
@@ -29,19 +38,17 @@ def create_nfse
...
-      errors.add(:bookings, 'não pode ficar em branco.')
-    end
-  end
-
   def set_reference
     self.update(reference: self.to_sgid(expires_in: nil).to_s)
   end
+
+  def duplicated
+    return unless Bill.billing_or_billed.includes(:bookings).pluck('bookings.id').reject(&:nil?).any? do |booking_id|
```
Comment (filabreu) [inline app/models/bill.rb:48]: :+1:

File: app/views/bookings/in_progress.html.erb [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <h1 class="title is-size-3 mt-5 has-text-centered">Serviços em Andamento</h1>
 
 <div class="buttons is-centered">
-  <%= link_to "Voltar para Clientes", customers_path, class: "button is-link" %>
+  <%= link_to "Voltar para Clientes" , customers_path, class: "button is-link" %>
```
Comment (filabreu) [inline app/views/bookings/in_progress.html.erb:4]: You can remove the extra space before the comma

File: app/views/bookings/in_progress.html.erb [hunk 3]
```diff
@@ -70,11 +79,11 @@
       <%= form.check_box(:is_gift, {}, true, false) %>
     </div>
 
-    <%= form.label "Aplicar desconto", class: "label mr-2" %>
-    <div class="field"><%= form.number_field :discount, class: "field input", in: 1..100 %></div>
-    <%= form.label "%", class: "label" %>
-
-    <%= form.submit "Finalizar Serviços", class: "button is-primary ml-2" %>
-
+    <%= form.label "Aplicar desconto" , class: "label mr-2" %>
```
Comment (filabreu) [inline app/views/bookings/in_progress.html.erb:82]: You can remove the extra space before the comma

File: spec/models/bill_spec.rb [hunk 4]
```diff
@@ -4,32 +4,56 @@
...
-    context "attributes" do
-      subject { invalid_bill.errors.attribute_names }
+  # before do
+  #   allow_any_instance_of(FocusNfeApi).to receive(:create)
+  #   allow_any_instance_of(FocusNfeApi).to receive(:get) do
+  #     {
+  #       status: "autorizado"
+  #     }
+  #   end
+  # end
```
Comment (filabreu) [inline spec/models/bill_spec.rb:20]: Let's make this mock work!

**Review by filabreu — CHANGES_REQUESTED** [review]

File: app/views/bookings/in_progress.html.erb (hunk 2 above)
Comment (rodolforpr88) [reply app/views/bookings/in_progress.html.erb]: Removed.

File: app/views/bookings/in_progress.html.erb (hunk 3 above)
Comment (rodolforpr88) [reply app/views/bookings/in_progress.html.erb]: Removed.

**Review by filabreu — APPROVED** [review]

**Pull request merged**